pages/bookmarks.jsx

```diff
@@ -14,57 +13,45 @@ export default React.createClass({
       bookmarks: null
     };
   },
-  verifyLoggedInStatus() {
-    Service.userstatus()
-      .then(response => this.updateLoggedInInfo(response.loggedin, response.username))
-      .catch(reason => {
-        this.updateLoggedInInfo(false);
-        console.error(reason);
-      });
-  },
-  updateLoggedInInfo(status, username = ``) {
-    UserData.userLoggedInStatus.set(status);
-    UserData.username.set(username);
-    this.forceUpdate();
-  },
-  logOutUser(event) {
+  handleLogOutBtnClick(event) {
     event.preventDefault();
-    Service.logout()
-      .then(() => {
-        // we have successfully logged user out
-        this.updateLoggedInInfo(false);
-      })
-      .catch(reason => {
-        console.error(reason);
-      });
+    Login.logoutUser(error => {
+      if (!error) {
+        this.forceUpdate();
```

Comment:
is this to make sure the page is reloaded in a state based on "now an anonymous user"?